nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/ExtensionBuilder.java

```diff
@@ -359,6 +363,19 @@ private void applyDefaultSettings(final ProcessorNode processorNode) {
         }
     }
 
+    private void applyDefaultRunDuration(final ProcessorNode processorNode) {
+        try {
+            final Class<?> procClass = processorNode.getProcessor().getClass();
+
+            final SupportsBatching sb = procClass.getAnnotation(SupportsBatching.class);
+            if (sb != null) {
+                processorNode.setRunDuration(sb.defaultDuration().getDuration().toMillis(), TimeUnit.MILLISECONDS);
+            }
+        } catch (final Exception ex) {
+            logger.error("Error while setting default run duration from SupportsBatching annotation: {}", ex.toString(), ex);
```

Comment:
Including the `ex.toString()` argument should not be necessary since the exception stack trace will include the same details.
```suggestion
            logger.error("Set default Run Duration failed", ex);
```